playbooks/roles/analytics_api/defaults/main.yml

```diff
@@ -20,28 +20,37 @@ ANALYTICS_API_PIP_EXTRA_ARGS: "-i {{ COMMON_PYPI_MIRROR_URL }}"
 ANALYTICS_API_NGINX_PORT: "18100"
 
 ANALYTICS_API_DEFAULT_DB_NAME: 'analytics-api'
+ANALYTICS_API_DEFAULT_DB_USER: 'api001'
+ANALYTICS_API_DEFAULT_DB_PASSWORD: 'password'
+ANALYTICS_API_DEFAULT_DB_HOST: 'localhost'
+ANALYTICS_API_DEFAULT_DB_PORT: 3306
 ANALYTICS_API_DEFAULT_MYSQL_OPTIONS: {}
+
 ANALYTICS_API_REPORTS_DB_NAME: 'reports'
+ANALYTICS_API_REPORTS_DB_USER: 'reports001'
+ANALYTICS_API_REPORTS_DB_PASSWORD: 'password'
```

Comment:
Is this something we can (and want to) make an assumption that we're reading secrets from secrets.yml?